.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
```

Comment:
Do we need this extension?

Reply:
No, good catch.

Reply:
For now I'm just following what we have on [Telescope](https://github.com/Seneca-CDOT/telescope/blob/master/.vscode/extensions.json). @humphd should I remove them?

Reply:
You can use Telescope as a guide, but this isn't a node/js project (e.g. we don't support the code, just the docs) so we don't need it here.